data/types/basal/basal_test.go

```diff
@@ -119,7 +120,7 @@ var _ = Describe("Basal", func() {
 			It("returns the expected identity fields", func() {
 				identityFields, err := datum.IdentityFields()
 				Expect(err).ToNot(HaveOccurred())
-				Expect(identityFields).To(Equal([]string{*datum.UserID, *datum.DeviceID, *datum.Time, datum.Type, datum.DeliveryType}))
+				Expect(identityFields).To(Equal([]string{*datum.UserID, *datum.DeviceID, (*datum.Time).Format(types.TimeFormat), datum.Type, datum.DeliveryType}))
```

Comment:
This test won't fail if `types.TimeFormat` is changed. I think it's best to duplicate the time format in the tests.